admin/schroot/Makefile

```diff
@@ -1,23 +1,25 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=schroot
-PKG_VERSION:=1.6.10
-PKG_RELEASE:=3
+PKG_NAME:=reschroot
+PKG_VERSION:=1.6.13
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).orig.tar.xz
-PKG_SOURCE_URL:=http://deb.debian.org/debian/pool/main/s/schroot
-PKG_HASH:=3ce8dfd9cb97b099e4b6d4ccec421d6cc8c9ef84574681e928a12badb5643d0b
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeberg.org/shelter/reschroot/archive/release
+PKG_HASH:=c05d6a1bb0210d401e6522eee2c465e0b43c6c98f7101d671f9e9ef88b3accf5
 
 PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
 PKG_LICENSE:=GPL-3.0-only
 PKG_LICENSE_FILES:=COPYING
 
-PKG_INSTALL:=1
-PKG_BUILD_PARALLEL:=1
-PKG_FIXUP:=autoreconf
+CMAKE_INSTALL:=1
+
+CMAKE_BINARY_SUBDIR:=build
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
```

Comment:
@neheb this is ok for you? If yes, then it can be merged.

Reply:
It is not.

Reply:
Maybe

```
PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)/$(PKG_NAME)
```